--- PHP_CodeSniffer-3.4.0/usr/share/pear/PHP/CodeSniffer/src/Filters/Filter.php~	2018-12-20 00:57:58.000000000 +0100
+++ PHP_CodeSniffer-3.4.0/usr/share/pear/PHP/CodeSniffer/src/Filters/Filter.php	2019-04-08 08:28:26.042280047 +0200
@@ -256,7 +256,7 @@
                 $testPath = $path;
             }
 
-            $pattern = '`'.$pattern.'`i';
+            $pattern = '`'.$pattern.'`';
             if (preg_match($pattern, $testPath) === 1) {
                 return true;
             }
